scripts/java-api-check-all.sh

```diff
@@ -25,8 +32,28 @@ if ! [ -x "$(command -v revapi.sh)" ]; then
     popd > /dev/null
 fi
 
-RELEATE_TAG=${1-""}
-echo "Release tag: $RELEATE_TAG"
+RELEASE_TAG=${1-""}
```

Comment:
we could move this code to extract last_version_tag to another script so this can be reused. but probably later.